app/views/manage_assessments/courses/_outcome_status.html.erb

```diff
@@ -4,7 +4,7 @@
     <h2 class="outcome-nickname"><%= outcome.nickname %></h2>
     <p class="outcome-card-description">
       <%= outcome.description %>
-    </p>
+    </p> 
```

Comment:
Can you get rid of this trailing whitespace here? Nothing major, obviously. I have vim configured to do this for me automatically on save. You might be able to do similar with an atom package: https://github.com/atom/whitespace